data/strings/types_strings.txt

```diff
@@ -21695,7 +21442,8 @@
     zh-Hant = 理髮師
 
   [type.shop.hardware]
-    ref = type.shop.doityourself
+    en = Hardware Store
+    en-GB = Ironmongers
```

Comment:
really?
I only sort of know en_GB, but I view Ironmonger as likely far far more specific than hardware.

Reply:
in GB, it refers to shops that sell diy supplies, tools, homeware, calor cannisters, door fittings, bike parts etc. just lots of useful stuff vaguely relating to maintenance of stuff.

> The term ironmonger as a supplier of consumer goods is still widely used in Great Britain, the US equivalent being "hardware store" - [Ironmongery on Wikipedia](https://en.wikipedia.org/wiki/Ironmongery)